RELEASING.md

```diff
@@ -76,7 +76,7 @@ and assign it to @bencochran or @aquageek.*
 1. If this was a fix release, merge changes to the master branch:
    ```bash
    git checkout master
-   git reset --hard origin/master
+   git pull
```

Comment:
ahh, good point! 

can we do:
```suggestion
   git fetch
   git reset --hard origin/master
```

This will protect us if we have some accidental commits in master.

Reply:
Reset hard will _lose_ accidental commits, pull will abort and ask to resolve. Also pull is just simpler, these instructions are complicated enough.

Reply:
Sgtm!